src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/OpprettFagsakModal.tsx

```diff
@@ -35,6 +35,7 @@ const OpprettFagsakModal: React.FC<IOpprettFagsakModal> = ({ lukkModal, deltager
                         type={'hoved'}
                         mini={true}
                         onClick={async () => {
+                            settSenderInn(true);
```

Comment:
Usikker på om vi skal sette denne her, eller om alt (altså sjekkTilgang) skal skje inne i opprett fagsak. Hva synes du @henninghaakonsen ? Det er i allefall for sent å sette den etter den await'en.

Reply:
Problemet her er egentlig at lukkModalen kalles rett etter opprettFagsak og ikke når den faktisk har returnert. Jeg tror dette er en løsning som vil skape bugs på sikt. Hva om man beholder logikken slik det var og gjør følgende:
1. Kaller settSenderInn(true) der du gjør.
2. Kun kaller lukkModal på then() fra opprettFagsak.

Da bør ikke modalen lukkes før man har opprettet fagsaken.

Reply:
Jeg kan kan kalle lukkModal på then() ja, det er nok lurt. Men modalen vil fortsatt lukke seg om man trykker hvor som helst på skjermen, så jeg tror det er lurt at man disabler at man får gjøre ting mens den holder på 🤔

Reply:
Godt poeng! 🙌